src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -1,17 +1,35 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
-  <PropertyGroup>
-    <PackageId>Monitor.Tests</PackageId>
-    <Description>Monitor.Tests Class library</Description>
-    <PackageId>Monitor.Tests</PackageId>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
-  </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Management.Monitor\Microsoft.Azure.Management.Monitor.csproj" />
-    <None Update="SessionRecords\**\*.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
-  </ItemGroup>
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+	<PropertyGroup>
+		<PackageId>Monitor.Tests</PackageId>
+		<Description>Monitor.Tests Class library</Description>
+		<PackageId>Monitor.Tests</PackageId>
+		<VersionPrefix>1.0.0-preview</VersionPrefix>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452</TargetFrameworks>
```

Comment:
@gucalder any reasons you are doing this.
If you are not able to discover tests that is because of VS 15.3 build.
I had sent an email about installing 15.2 and provided link to install 15.2.
Once you install 15.2 you should be good. Please revert all these changes in your test project.
These changes makes it difficult to make sure all SDKs are created and tested consistently with respect to target framework and updating common references.